Dockerfile

```
@@ -0,0 +1,31 @@
+# Copy files to install the dependencies
+COPY --chown=nonroot:nonroot package.json bun.lockb .
+
+# Install dependencies
+RUN bun install
+
+# Copy the rest of the application code
+COPY --chown=nonroot:nonroot ./public ./public
+COPY --chown=nonroot:nonroot ./src ./src
+COPY --chown=nonroot:nonroot ./tests ./tests
+
+COPY --chown=nonroot:nonroot ./eslint.config.js \
```

Hadolint

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)